lib/vsc/utils/rest.py

```diff
@@ -35,13 +35,20 @@
 @author: Jens Timmerman
 """
 import base64
-import urllib
-import urllib2
 try:
```

Comment:
remove the try/except, simplejson is from py24 days

Reply:
fixed